spec/05-classes-and-objects.md

```diff
@@ -435,13 +435,13 @@ An early definition is type-checked and evaluated in the scope which
 is in effect just before the template being defined, augmented by any
 type parameters of the enclosing class and by any early definitions
 preceding the one being defined. In particular, any reference to
-`this` in the right-hand side of an early definition refers
+`this` on the right-hand side of an early definition refers
 to the identity of `this` just outside the template. Consequently, it
 is impossible that an early definition refers to the object being
 constructed by the template, or refers to one of its fields and
```

Comment:
Both "refers" could be subjunctive "refer", as the condition is explicitly counterfactual. "...or that it refer..." The tone would be rather high.